internal/exec/vendor_model.go

```diff
@@ -170,17 +173,16 @@ func (m *modelVendor) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			)
 		}
 		if !m.isTTY {
-			u.LogInfo(fmt.Sprintf("%s %s %s", mark, pkg.name, version))
+			log.Info(fmt.Sprintf("%s %s %s", mark, pkg.name, version))
```

Comment:
It was wrong in the original implementation, and it's wrong here. We don't "log" TUI console output, even if there's no TTY. We'll fix this in a new PR.